lms/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -318,3 +318,46 @@ def test_get_view_for_dop(self):
     def test_get_view_for_no_backend(self):
         view_object = views.AccessTokenView()
         self.assertRaises(KeyError, view_object.get_view_for_backend, None)
+
+
+class TestRevokeTokenView(_DispatchingViewTestCase):  # pylint: disable=abstract-method
+    """
+    Test class for RevokeTokenView
+    """
+
+    def access_token_post_body(self, user, client):
+        """
+        Returns a dictionary to be used as the body of the access_token POST request
+        """
+        return {
+            'client_id': client.client_id,
+            'grant_type': 'password',
+            'username': user.username,
+            'password': 'test',
+        }
+
+    def revoke_token_post_body(self, client, token):
+        """
+        Returns a dictionary to be used as the body of the revoke_token POST request
+        """
+        return {
+            'client_id': client.client_id,
+            'client_secret': client.client_secret,
+            'token': token,
+        }
+
+    def test_revoke_token_view_dot(self):
```

Comment:
Can you add 1 additional test for revoking an **access** token?  The specs allow for it, as does the implementation in DOT.   So we might as well make sure it works.